analytics/tests/conftest.py

```diff
@@ -21,6 +22,28 @@
 DAY_5 = "2023-11-05"
 
 
+class MockSlackbot:
+    """Create a mock slackbot issue for unit tests."""
+
+    def upload_files_to_slack_channel(
+        self,
+        channel_id: str,
+        files: list,
+        message: str,
+    ) -> None:
+        """Stubs the corresponding method on the main SlackBot class."""
+        assert isinstance(channel_id, str)
+        print("Fake posting the following files to Slack with this message:")
```

Comment:
Confirming we want this printed, and not logged?

Reply:
I think we want printing in the following scenarios:

- Printing to the console from the CLI
- Printing within tests

This this mock `post_to_slack()` method is used in tests I think the use of `print` might be okay here. 

I did create this task for us to [track adding logging](https://github.com/HHS/simpler-grants-gov/issues/918) and we can discuss further when work begins on this issue.